notebooks/notebooks/paper/katago-new-ckpt-evals.ipynb

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.                ci_df.upper,  # type: ignore
</pre><p> we generate Clopper-Pearson intervals in a couple other notebooks in this repo and I don't remember needing <code># type: ignore</code> — is this notebook doing something slightly different, or did we just not bother running pytype on the other notebooks? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Ah I think my vscode editor is just super aggressive with typing. It marks other notebooks with red squiggles when you don't to type ignore.

For example in https://github.com/AlignmentResearch/KataGoVisualizer/blob/main/notebooks/notebooks/paper/baseline-attacks.ipynb.
<img width="676" alt="image" src="https://user-images.githubusercontent.com/1733389/232251635-cd48dba9-708e-438c-8241-87cbaaee67f6.png">